CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 -   Allow interfaces in classes.
 -   Python support for a simple class.
 -   #1430 : Added conjugate support to booleans.
+-   #1452 : Added C support for a class containing only functions
```

Comment:
I think it would be better to mention "printing" here in case someone thinks that we added support for classes (which would require the wrapper to be implemented too)
```suggestion
-   #1452 : Added C printing support for a class containing only functions
```

Reply:
Fixed